### PR TITLE
i18n Led Strip and others

### PR DESCRIPTION
**Pull request by McGiverGim**
More i18n, specially the led tab. I've tested it and all seems ok.

### CODE REVIEW

File: locales/en/messages.json [hunk 1]
```diff
@@ -2311,15 +2474,27 @@
         "message": "deep pink"
     },
     "controlAxisRoll": {
-        "message": "Roll [A]"
+        "message": "Roll"
```
Comment (mikeller) [inline locales/en/messages.json]: Not sure if this should be done. `AETR` seem to be the commonly used designators for the axes in fixed wing flying.

Comment (McGiverGim) [reply locales/en/messages.json]: I didn't know, I thought these were shortcuts for the MSP virtual remote... I will change it.

Comment (mikeller) [reply locales/en/messages.json]: They stand for [A]ileron, [E]levator, [T]rottle, [R]udder, which are well defined in fixed wing aircraft.

**Comment by McGiverGim** [comment]
Changed according review. Need some test, but it seems to work.

**Pull request merged**